sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -155,12 +155,16 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             timeout=cdk.Duration.seconds(60)
         )
 
-        lambda_.FunctionUrl(self,
-            id="DownloadQueryAPI",
-            function=download_query_api,
-            auth_type=lambda_.FunctionUrlAuthType.NONE,
-
-            cors=lambda_.FunctionUrlCorsOptions(
-                                allowed_origins=["*"],
-                                allowed_methods=[lambda_.HttpMethod.GET])
+        download_api_url = lambda_.FunctionUrl(self,
+                                               id="DownloadQueryAPI",
+                                               function=download_query_api,
+                                               auth_type=lambda_.FunctionUrlAuthType.NONE,
+                                               cors=lambda_.FunctionUrlCorsOptions(
+                                                                   allowed_origins=["*"],
+                                                                   allowed_methods=[lambda_.HttpMethod.GET])
         )
+########### OUTPUTS
+        # This is a list of the major outputs of the stack
+        #cdk.CfnOutput(self, "UPLOAD_API_URL", value=upload_api_url.url)
+        cdk.CfnOutput(self, "QUERY_API_URL", value=lambda_query_api_function_url.url)
```

Comment:
What are these outputs used for? From googling, it looks like they have a few different uses.

Reply:
They don't do anything directly within the stack, but after the stack is finished it prints out the values of the generated URLs.  These values are saved in AWS, so you can easily reference the values again from elsewhere.  For example, other Cloudformation stacks can reference these outputs.  So you can link cloudformation stacks in a chain, feeding the output of one stack into the input of another.  

Really the only thing I'm using it for now is so I can copy+paste those URLs into my test script without inspecting the AWS console to see what URLs AWS came up with this time.  But we might have more use for these outputs in the future.

Reply:
I didn't think of doing that. Really makes it easy to look up these values like you said.

Reply:
Alright, great! So right now just for testing, but maybe in the future, to be used in other ways. Thanks for explaining!